src/DynamoFeatureFlags/DynamoFeatureFlags.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <ImportGroup Label="PropertySheets">
+    <Import Project="$(SolutionDir)Config/CS_SDK.props" />
+  </ImportGroup>
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <!--force copying nuget netstd2 dependencies-->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="LaunchDarkly.ClientSdk" Version="2.0.1">
+      <!--using this to avoid copying net461 deps of launch darkly -->
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+    <!--to force LD package to reference system.collections.immutable of the same version. -->
+    <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
```

Comment:
I am not sure how well `CopyLocalLockFileAssemblies` will work vs same pacakgeReferences in other projects. 
Ex CopyLocalLockFileAssemblies = true will pull down the dotnetstandard flavor of all pacakgeReferences in the csproj. But there are other Dynamo projects which could have the same pacakgeRefereces (but with no copylocallock setting)
ex. 
1. System.Collections.Immutable@5.0.0 referenced in featureFlags csproj will copy over System.Collections.Immutable.dll from the netstandard2.0 subfolder
2. System.Collections.Immutable@5.0.0 referenced from designscriptbuiltin would copy over System.Collections.Immutable.dll from net461 (closest target framework moniker to the projects tfm)